main.py

```diff
@@ -14,6 +15,9 @@
 
 if __name__ == '__main__':
 
+    message = os.getenv('INPUT_COMMITMESSAGE')
+    print(f'Commit message: {message}')
```

Comment:
I wondered if this should be part of the message below?  Or after it?  This is quite possibly too trivial a comment :)

Reply:
"After it" feels more natural.